### DISCUSSION TITLE
Idle timer not working

### DISCUSSION THREAD

**Discussion by luisete79** [MiczFlor/RPi-Jukebox-RFID#2393] [Q&A]
Hi all. My phoniebox is working great. I burned an image of my SD card into a bigger and faster microSD card. However now the idle timer shutdown is not starting and it appears always as non idle.
The thing is that it used to work before, and I have not changed anything (other than saving an image of my card and burn it again on a different card).
Is there any way to fix de idle countdown?
Thanks a lot for your help!
(I am using  the last version of v.2 on a RPi 3B)

**Comment by pabera** [comment]
I am not sure why it's not working now, you could check the logs if you like. 
Nevertheless, there is a new PR related to this https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/2332 and I hope I can release it soon.

**Reply by luisete79** [reply]
Thanks your reply. I'm kind of a beginner, sorry. Where are the logs located?

**Reply by s-martin** [reply]
For V2 please use the command `tail -n 200 /var/log/syslog`

**Comment by luisete79** [comment]
Thanks! This might be it, happening every minute:
Jul  3 20:58:22 raspberrypi idle-watchdog.sh[10684]: warning: commands will
be executed using /bin/sh
Jul  3 20:58:22 raspberrypi idle-watchdog.sh[10684]: Cannot create atjob
file /var/spool/cron/atjobs/i0072401b56d5c: Permission denied
Jul  3 20:59:24 raspberrypi idle-watchdog.sh[11252]: warning: commands will
be executed using /bin/sh
Jul  3 20:59:24 raspberrypi idle-watchdog.sh[11252]: Cannot create atjob
file /var/spool/cron/atjobs/i0072501b56d5d: Permission denied

**Reply by luisete79** [reply]
Hi all, 
I updated my phoniebox to v. 2.8, just to see if this solved the problem, but it didn't. Anyone knows how to get the idle-watchdog.sh to work?
Thanks a lot in advance

**Comment by luisete79** [comment]
Hi,
Looks like if the cloned card does another funny thing: the music is often
paused or resumed when I switch on or off the lights of the room. This does
not bother me too much (I can always press the button to pause or resume
again) but I would really want to fix the idle shutdown. Does anyone know
how I could do it?
Thanks in advance
